### PR TITLE
Mini player change

### LINKED ISSUE #556: Mini player change
For accessibly and based on feedback, we need to modify the mini player and the behaviour around it. 

You can find the design here:
[https://www.figma.com/file/E9k4sKjlBDUREveJbUtf4A/Medito-2023?type=design&node-id=2203%3A12492&mode=design&t=c76gIkS5uCkx8Rf0-1](https://www.figma.com/file/E9k4sKjlBDUREveJbUtf4A/Medito-2023?type=design&node-id=2203%3A12492&mode=design&t=c76gIkS5uCkx8Rf0-1)

The behaviour is also indicated on Figma. 

- [x] Remove the page view behaviour
- [x] Change the mini player for a sticky one
- [x] Make sure the mini player handles the case of with artist and no artist + marquee
- [x] Allow the mini player to be dismiss when paused
- [x] Automatically remove the mini player when the audio finishes
- [x] Make the big player global and allow to open it by tapping on the mini player or by swapping up on the mini player
- [x] [canceled] Make the device nav (iOS + android) bar with the Onyx color

Ping me if you need more info :)

### PR DESCRIPTION
**Pull request by osamaasifoak**
[Mini player change#556](https://github.com/meditohq/medito-app/issues/556)

### CODE REVIEW

File: lib/views/root/root_page_view.dart [hunk 1]
```diff
@@ -64,63 +65,23 @@ class _RootPageViewState extends ConsumerState<RootPageView> {
...
-                  ),
+                _renderChild(
+                  context,
+                  connectivityStatus as ConnectivityStatus,
                 ),
-                _miniPlayer(radius, currentlyPlayingSession),
+                StickyMiniPlayerWidget(),
               ],
             ),
-            if (currentlyPlayingSession != null)
```
Comment (michaelcspeed) [inline lib/views/root/root_page_view.dart:84]: can delete this comment

**Review by michaelcspeed — APPROVED** [review]
@osamaasifoak I fixed the conflicts - over the weekend I created a new top bar view that collapses with a nice fade, so not sure if this one also needs this new BottomPaddingWidget. I will merge this but please can you check the padding? And also remove this commented out line. Cheers :)

**Comment by michaelcspeed** [comment]
Just realised I can actually edit files in github! So I removed the comment myself :)

File: lib/views/root/root_page_view.dart (hunk 1 above)
Comment (osamaasifoak) [reply lib/views/root/root_page_view.dart:84]: It's already removed

**Comment by osamaasifoak** [comment]
Yes we still need this BottomPaddingWidget.

**Pull request merged**